src/index.tsx

```diff
@@ -23,6 +23,7 @@ const root = createRoot(container as Element);
 
 Sentry.init({
   dsn: process.env.PUBLIC_SENTRY_DSN,
+  environment: process.env.PUBLIC_ENVIRONMENT,
```

Comment:
@SovereignAndrey, to add to `BG-1191:staging: sentry error should only be captured in production`
in `helpers/logger.ts`

`logger.error` won't log in production (`IS_TEST = false`)
```js
export const logger: Logger = IS_TEST
  ? {
      // using console.log for .error and .info funcs as console.error and
      // console.info are undefined while testing for some reason
      error: (message) => {
        Sentry.captureException(message);
        console.log(message);
      },
      info: (...data) => console.log(...data),
      log: (...data) => console.log(...data),
    }
  : {
      error: (_) => {},
      info: (_) => {},
      log: (_) => {},
    };

```
should be changed to 
```js
export const logger: Logger = IS_TEST
  ? {
      // using console.log for .error and .info funcs as console.error and
      // console.info are undefined while testing for some reason
      error: (message) => {
        console.log(message);
      },
      info: (...data) => console.log(...data),
      log: (...data) => console.log(...data),
    }
  : {
      error: (_) => {
         Sentry.captureException(message); //move sentry capture here
      },
      info: (_) => {},
      log: (_) => {},
    };

```

Reply:
@ap-justin Yeah, that's true with regards to flooding sentry with staging/dev errors. Hmm... :thinking: 
I guess we're safe to capture _only_ production level changes in Sentry. We already have dev logging to debug local issues & staging app bugs that Operations catches can always be triaged internally. Will make those needed changes.